src/utils/customComponents.js

```diff
@@ -0,0 +1,9 @@
+import ProseBlockquote from '../components/ProseBlockquote.vue';
```

Comment:
This is where we're defining the custom components that we want Astro to override the default styling of our MDX components 

It is utilised in `src/pages/guides/[...slug.astro]`